app/src/main/java/com/twilio/video/app/util/CameraCapturerCompat.kt

```diff
@@ -0,0 +1,105 @@
+package com.twilio.video.app.util
+
+import android.content.Context
+import android.graphics.ImageFormat
+import android.hardware.camera2.CameraCharacteristics
+import android.hardware.camera2.CameraManager
+import android.hardware.camera2.CameraMetadata
+import android.os.Build
+import com.twilio.video.Camera2Capturer
+import com.twilio.video.CameraCapturer
+import com.twilio.video.VideoCapturer
+import timber.log.Timber
+import tvi.webrtc.Camera1Enumerator
+import tvi.webrtc.Camera2Enumerator
+import tvi.webrtc.CameraEnumerator
+
+class CameraCapturerCompat(
```

Comment:
Have this subclass `VideoCapturer` and you will not need to make the `videoCapturer` property public

Reply:
Ah nice good call. Also, It's a bit annoying having to have two properties to represent cameraCapturer and camera2Caputer. It would be nice if they both extended a common type

Reply:
Oh sorry. I meant it'd be nice if `getCameraId` was part of a common type so that way only a single property would be required to house the CameraCapturer